rialto_airflow/mais.py

```diff
@@ -2,7 +2,7 @@
 import logging
 from collections import defaultdict
 from datetime import date
-from typing import Any, Dict, List, Optional, Union
+from typing import Any, Optional, Union
```

Comment:
turns out `typing.Dict` and `typing.List` are deprecated in favor of plain `dict` and `list` (nothing complained about that, i stumbled on it in the docs)

Reply:
Good to know, I've seen both styles and was wondering which was preferred.